src/scenes/Languages/Detail/LanguageDetail.tsx

```diff
@@ -137,7 +150,21 @@ export const LanguageDetail = () => {
                 </Fab>
               </Tooltip>
             ) : null}
+            {!language ? (
+              <Skeleton width="100%" />
+            ) : (
+              <Tooltip title="This indicates the project/language(s) will be exposed to major investors to directly fund.">
+                <IconButton className={classes.presetInventoryIcon}>
```

Comment:
Not a button here, only an icon. I'd probably just only show the filled icon if true and omit if false.